src/api/app/views/webui2/webui/staging_workflows/_infos.html.haml

```diff
@@ -1,4 +1,17 @@
 .card.mb-3
   %h5.card-header Infos
   .card-body
-    Placeholder for the content
+    %dl
+      %dt Empty projects:
+      %dd= render 'empty_projects_list', projects: empty_projects
+
+      %dt Backlog:
+      %dd= render 'requests_list', requests: unassigned_requests, more_requests: more_unassigned_requests
+
+      %dt= link_to('Ready:', project_requests_path(project, state: 'new'))
```

Comment:
This is a tricky one - because previously 'new' meant ready - but now you consider 'new' as unassigned. Please leave a TODO here - this not yet done decision about the role of staging manager affects both ready backlog as ready.

Reply:
I will leave this as a header, no link. Until we clarify this.